labs/intro.md

```diff
@@ -0,0 +1,111 @@
+## Lab exercise 0: Setup steps
+
+:imagesdir: images
+
+### Accessing your dedicated lab environment using your provided GUID
+
+On your Red Hat Summit provided laptop, notice that the https://www.opentlc.com/gg/gg.cgi?profile=generic_summit[*Lab GUID Assignment page*^] is already loaded onto your Firefox web browser.
```

Comment:
Aren't people going to bring their own devices? I thought they said during the meeting today that devices won't be provided

Reply:
oh yea, i copied that from an existing lab.  I'll fix.